site/.vuepress/enhanceApp.js

```diff
@@ -0,0 +1,163 @@
+const oldUrl= [
+    '/software/',
```

Comment:
Given that old and new URLs are highly related to each other i.e., a person reading one version has a high chance of wanting to read the other, ideally, we want to keep them together.

How about changing to the following? I see that adds a little of repetition, but in this case, I see as a positive one. It facilitates the understanding of the redirections.

```javascript
const redirectList = [
  {
    path: "/software/",
    redirect: "/products/"
  },
  {
    path: "/articles/",
    redirect: "/blog/"
  }
];
```